### PR TITLE
Fix bump_recipes_version and some recipes

### PR DESCRIPTION
**Pull request by martinRenou**

### CODE REVIEW

**Comment by martinRenou** [comment]
@DerThorsten I'm getting `emmake: error: config file not found: /home/runner/work/recipes/recipes/emsdk/upstream/emscripten/.emscripten.  Please create one by hand or run emcc --generate-config`, but I'm doing so in my latest commit. Do you know what could be wrong here?

**Comment by DerThorsten** [comment]
> @DerThorsten I'm getting `emmake: error: config file not found: /home/runner/work/recipes/recipes/emsdk/upstream/emscripten/.emscripten. Please create one by hand or run emcc --generate-config`, but I'm doing so in my latest commit. Do you know what could be wrong here?

https://github.com/emscripten-forge/recipes/pull/325 this might do the trick

**Comment by DerThorsten** [comment]
with #325 merged this should work

**Comment by jtpio** [comment]
Thanks!

Looks like it should fine to merge and try the script again?

File: recipes/recipes_emscripten/arrow-python/recipe.yaml
Comment (DerThorsten) [inline recipes/recipes_emscripten/arrow-python/recipe.yaml:1]: I think we need to update the build number here

Comment (DerThorsten) [reply recipes/recipes_emscripten/arrow-python/recipe.yaml:1]: at least the CI will try to upload this package with a build number / version which already exists

File: recipes/recipes_emscripten/clapack/recipe.yaml
Comment (DerThorsten) [inline recipes/recipes_emscripten/clapack/recipe.yaml:1]: probably also need to increment build number

File: recipes/recipes_emscripten/openssl/recipe.yaml
Comment (DerThorsten) [inline recipes/recipes_emscripten/openssl/recipe.yaml:1]: build number?

**Review by DerThorsten — CHANGES_REQUESTED** [review]

File: recipes/recipes_emscripten/libffi/recipe.yaml [hunk 1]
```diff
@@ -2,46 +2,20 @@ context:
...
-    - sel(emscripten): texinfo
-    - sel(win):  autotools_clang_conda            
-    - sel(unix or emscripten): make                             
-    - sel(unix): autoconf                         
-    - sel(unix): libtool                          
-    - sel(unix): patch                            
-    - sel(unix): automake      
-    - sel(win):  m2-textinfo
-    - sel(unix and not emscripten): texinfo
-    - sel(unix or emscripten): '{{ compiler("cxx") }}'
```
[the remark this answers is not in the document]
Comment (martinRenou) [reply recipes/recipes_emscripten/libffi/recipe.yaml:41]: this was resolved already

**Comment by martinRenou** [comment]
thks

**Review by DerThorsten — APPROVED** [review]

**Pull request merged**